homedecor_bathroom/init.lua

```diff
@@ -99,12 +147,21 @@ homedecor.register("medicine_cabinet", {
 	inventory = {
 		size=6,
 	},
+	crafts = {
+		{
+			recipe = {
+				{ "group:stick", "glass_block", "group:stick" },
+				{ "group:stick", "glass_block", "group:stick" },
+				{ "group:stick", "glass_block", "group:stick" }
```

Comment:
do you mean
```lua
				{ "group:stick", glass_block, "group:stick" },
				{ "group:stick", glass_block, "group:stick" },
				{ "group:stick", glass_block, "group:stick" }
```

?

i see this same mistake in other files as well

Reply:
no, i do not

Reply:
since everything is registered through homedecor.register https://github.com/mt-mods/homedecor_modpack/blob/game_agnostic/homedecor_common/registration.lua#L99-L124 handles that

Reply:
i see. terrible idea, but at this point i'm just shrugging at the huge quantity of bad decisions in this PR. i know it's not all your fault.